ui-tests/src/test/java/org/openmrs/reference/RecordMergeIssueTest.java

```diff
@@ -80,6 +83,5 @@ public void tearDown() throws Exception {
         p.uuid = patient.uuid;
         deletePatient(p);
         waitForPatientDeletion(patient.uuid);
-        headerPage.logOut();
```

Comment:
After deleting the test patient data from the `db instance` there is no need to invoke `headerPage.logOut()`  within `tearDown() ` function

Reply:
Why not Adding header.logout, do you mean after deletion of patient, and the test wait to delete a patient  the testpasses without logging out ??

Reply:
> Why not Adding header.logout

This being a `teardown` function, I think there is no need to invoke `headerPage.logout()`. Basing on the legacy ui test `workflow`, the test comes to an end when the `test patient data` has been deleted from `db instance` by the logic of `RestClient`. By the time   `headerPage.logOut()` is invoked, the` test patient data` has been already deleted from the `db instance` at  LN 84 & 85


> do you mean after deletion of patient, and the test wait to delete a patient the testpasses without logging out ??

After simulation on the test without invoking  `headerPage.logOut()` within the `teardown` function, the test `passes`.